install/linux/usr/share/odemis/hwtest/tescan-fibsem.odm.yaml

```
@@ -0,0 +1,85 @@
+SEM: {
+    class: Microscope,
+    role: sem,
+}
+
+# Connects to Tescan SEM via SharkSEM
+"Tescan SEM": {
+    class: tescan.SEM,
+    role: null,
+    init: {host: "192.168.1.181"},
+    # more detectors can be added, if necessary
+    children: {
+        stage: "Sample Stage",
+        # SEM
+        scanner: "EBeam Scanner",
+        detector0: "EBeam Detector",
+        focus: "EBeam Focus",
+        # FIB
+        fib-scanner: "Ion Scanner",
+        fib-detector: "Ion Detector",
+        fib-focus: "Ion Focus",
+        # Chamber
+        camera: "Chamber Cam", # comment out if using the simulator
+        light: "Chamber Light",
+    }
+}
+
+"Sample Stage": {
+    role: stage,
+    init: {},
+    affects: ["EBeam Scanner"]
+}
+
+"EBeam Scanner": {
+    # Internal child of Tescan, so no class
+    role: e-beam,
+    init: {
+        fov_range: [196.e-9, 25586.e-6]
+    },
+}
+
+"EBeam Detector": {
+    role: se-detector,
+    init: {
+        channel: 0,
+        detector: 1,
+    },
+}
+
+"EBeam Focus": {
+    role: ebeam-focus,
+    init: {axes: [z]},
+    affects: ["EBeam Scanner"]
+}
+
+# FIB specific components
+
+"Ion Scanner": {
+    role: ion-beam,
+    init: {}
+}
+
+"Ion Detector": {
+    role: "se-detector-ion",
```

```suggestion
    role: "ion-detector",
```

Not sure about the naming